recipes/cppyythonizations/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "cppyythonizations" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/flatsurf/cppyythonizations/releases/download/{{ version }}/cppyythonizations-{{ version }}.tar.gz
+  sha256: 94cf064dc6c5f048b5987fb9dab295700403255ab993ff9b29cda9f3a9629517
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  build:
+    - automake
+    - coreutils
+    - make
```

Comment:
```suggestion
    - make
    - {{ compiler('cxx') }}
```

Reply:
I don't think there's a need for a compiler: https://github.com/flatsurf/cppyythonizations/blob/master/src/setup.py.in

Reply:
Ah, I was just basing it off the recipe in the upstream repo.

So `make`, `automake`, and `coreutils` are only needed to get things installed in the right spot?

If the stuff is built at runtime, then you're going to need some kind of compiler under `run`.

Reply:
The compiler is pulled in by cppyy actually which is a runtime dependency.

Reply:
> So `make`, `automake`, and `coreutils` are only needed to get things installed in the right spot?

Yes. This project is originally from a stack of projects that are all written with autotools. Most of them are compiled C++ projects but this one is not.